src/main/java/fixie/dictionary_service/controller/DictionaryController.java

```diff
@@ -0,0 +1,90 @@
+package fixie.dictionary_service.controller;
+
+import fixie.dictionary_service.dto.ActivityDictionaryDTO;
+import fixie.dictionary_service.dto.PartTypeDTO;
+import fixie.dictionary_service.entity.ActivityDictionary;
+import fixie.dictionary_service.entity.PartType;
+import fixie.dictionary_service.service.DictionaryService;
+import fixie.user_service.dto.UserDTO;
+import lombok.SneakyThrows;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+public class DictionaryController {
+
+
+    private final DictionaryService dictionaryService;
+
+    public DictionaryController(DictionaryService dictionaryService) {
+        this.dictionaryService = dictionaryService;
+    }
+
+
+    @SneakyThrows
```

Comment:
I told you how does SneakyThrows annotation work. Use brain when you copy&paste code. In your current implementation there isn't ANY method, with throws an exception (there are some that should throw, but i'll leave it to you to figure it out which ones), so why do you use @SneakyThrows?

Reply:
I tried to add an exception but then found it unnecessary and did not remove everything.

Reply:
fixed